cdap-cli/src/main/java/co/cask/cdap/cli/command/system/HelpCommand.java

```diff
@@ -225,16 +225,22 @@ private void wrappedPrint(String str, PrintStream output, int colWidth, int pref
     }
 
     int beginIdx = 0;
+    int returnIdx = 0;
```

Comment:
I think ``\n`` is widely understood as a newline character. So ``newLineCharIndex``?